ic-os/setupos/config/config.ini

```diff
@@ -1,10 +1,19 @@
 # Network Configuration File
 #
-# Update these settings according to your specific network requirements.
+# Update these settings according to your specific network and node requirements.
 
+# ------------------------------
+# Node Remuneration Settings (required)
+# ------------------------------
+# Update node_remuneration_type as per your node configuration.
+# Node remunerations should be of the form:
+# node_remuneration_type=typeX or node_remuneration_type=typeX.Y
+# Example:
+# node_remuneration_type=type2.1
```

Comment:
@sasa-tomic Is 2.1 the most 'standard' remuneration type?

Reply:
Actually it's type3.1 - the latest and greatest for gen2 node providers. The grand majority of node providers are gen1 though, and they are type1. Gen1 will all have to switch to type1.1 in the next 6 months or so.

Reply:
Updated to 3.1!